protocol/protobuf/application_metadata_message.proto

```diff
@@ -68,7 +68,6 @@ message ApplicationMetadataMessage {
     DECLINE_CONTACT_VERIFICATION = 52;
     SYNC_TRUSTED_USER = 53;
     SYNC_VERIFICATION_REQUEST = 54;
-    CONTACT_VERIFICATION_TRUSTED = 55;
```

Comment:
Should i update other values starting from SYNC_CONTACT_REQUEST_DECISION?

Reply:
@MishkaRogachev maybe it's better to mark it as deprecated:
`CONTACT_VERIFICATION_TRUSTED = 55 [deprecated=true];`